src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+import React, {useEffect} from 'react';
+import styled, {ThemeProvider} from 'styled-components';
+import {AppTheme, lightTheme} from '@datorama/app-components';
+import {Provider, useDispatch, useSelector} from 'react-redux';
```

Comment:
usually we separate the components and other stuff. for example - 

// UTILS

// COMPONENTS

// TYPES